app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CategoriesController < ApplicationController
+
+  def new
+    @category = Category.new
+  end
+
+  def create
+    @category = Category.new(category_params[:category])
+    @category.name.capitalize!
```

Comment:
Nice touch for standardizing your categories!

Especially in conjunction with the uniqueness validation in the model.